code/modules/antagonists/heretic/heretic_antag.dm

```diff
@@ -765,8 +777,8 @@
 	target_amount = main_path_length
 	// Add in the base research we spawn with, otherwise it'd be too easy.
 	target_amount += length(GLOB.heretic_start_knowledge)
-	// And add in some buffer, to require some sidepathing.
-	target_amount += rand(2, 4)
+	// And add in some buffer, to require some sidepathing. // Increased by 3 now that each path gets 3 free side points.
```

Comment:
```suggestion
	// And add in some buffer, to require some sidepathing, especially since heretics get some free side paths.
```
Comments shouldn't be dated